From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@animetauren__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@cherylmc - Could you please review this public repo contribution?

cherylmc commented:
@animetauren - thanks for adding to this. Appreciate it! @tysonn - could you please merge this one in?

mattbriggs committed:
stubbing out topics

mattbriggs committed:
update links

mattbriggs committed:
adding info

mattbriggs committed:
adding CLI

mattbriggs committed:
update api versions

mattbriggs committed:
adding files

mattbriggs committed:
updating profile material

Brenduns committed:
asdk edits for removed notes

Brenduns committed:
Fixing merge issues

Brenduns committed:
Update asdk-release-notes.md

Fixing a minor formatting issue.

Brenduns committed:
Update asdk-release-notes.md

Fixed issues.

Brenduns committed:
Update azure-stack-update-1803.md

Removed commented section, it was just a placeholder for others contributing to this content.

Ja-Dunn committed:
Merge pull request #35811 from Brenduns/release-1803

Release 1803

mattbriggs committed:
adding updated code samples

v-alje committed:
Bringing even with master.

mattbriggs committed:
adding new items

SHEETHALJS committed:
(ForumsTeam)-broken links 

edited two broken links for groups

mrbullwinkle committed:
Test Build of pricing page DO NOT MERGE BEFORE 4/2/2018

mrbullwinkle committed:
Add bad link to temporarily prevent merge

mrbullwinkle committed:
small grammar + spacing corrections

Brenduns committed:
updates for release 1803

v-alje committed:
Bringing even with master.

mrbullwinkle committed:
split out details for enterprise pricing + TOC mod

mrbullwinkle committed:
small grammar fix

mrbullwinkle committed:
Fix link to details doc

Brenduns committed:
updates for release 1803

mrbullwinkle committed:
Fix links

rmca14 committed:
Merge pull request #35918 from Brenduns/release-1803

updates for release 1803

mrbullwinkle committed:
add changes as per Dale's feedback

mrbullwinkle committed:
small changes per Dale

v-alje committed:
Bringing even with master.

Brenduns committed:
Update for Azure Monitor

Brenduns committed:
Release notes sync for ASDK and IS on march 22

Brenduns committed:
Release notes sync for ASDK and IS on march 22

v-alje committed:
Bringing even with master.

Brenduns committed:
Fixing merge conflict (on the article date)

Brenduns committed:
Cleaningup two ASDK items

Ja-Dunn committed:
Merge pull request #36042 from Brenduns/release-1803

Release 1803

v-alje committed:
Bringing even with master.

mattbriggs committed:
update code snippets

mattbriggs committed:
updating code snippets

mrbullwinkle committed:
Image updates and a few text changes

mrbullwinkle committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into mrb_03_21_2018_pricing

mattbriggs committed:
update TOC

mattbriggs committed:
update command

v-alje committed:
Bringing even with master.

mattbriggs committed:
clear up mulched text

Brenduns committed:
updates for release notes

Brenduns committed:
updates for release notes

Brenduns committed:
updates for release notes

Brenduns committed:
updates for release notes

Brenduns committed:
ASDK revisions

Brenduns committed:
ASDK revisions

PRMerger16 committed:
Merge pull request #36174 from Brenduns/release-1803

Release 1803

v-alje committed:
Bringing even with master.

kadimitr committed:
Extending the documentation to include clarification on a new feature implemented with the following feature - https://github.com/Azure/azure-functions-durable-extension/issues/71

Brenduns committed:
Relnote revisions, and offer revision.

Brenduns committed:
Relnote revisions, and offer revision.

Brenduns committed:
udpate for jeff

v-alje committed:
Bringing even with master.

Brenduns committed:
Merge branch 'release-azs-1803' into release-1803

mattbriggs committed:
adding updates to the secrets

mattbriggs committed:
update PKI:

mattbriggs committed:
updating formatting issues

GitHubber17 committed:
capitalization for product name - PowerShell

Ja-Dunn committed:
Merge pull request #36244 from Brenduns/release-1803

Relnote revisions, and offer revision.

GitHubber17 committed:
Merge pull request #36262 from mattbriggs/mdb_new_azs_1803

Adding Rotate Secrets

paulathurman committed:
edit pass: concepts-pricing-tiers

paulathurman committed:
edit pass: concepts-pricing-tiers

paulathurman committed:
edit pass: concepts-pricing-tiers

mattbriggs committed:
updating item

paulathurman committed:
edit pass: concepts-pricing-tiers

mattbriggs committed:
adding update

v-alje committed:
Bringing even with master.

kadimitr committed:
Consolidating sample code for Single instance query to the one containing ordering.

v-mavick committed:
Corrected broken link to Free Azure subscription page.

Commit:
Last minute 1803 feedback

Commit:
Last minute 1803 feedback

v-alje committed:
Bringing even with master.

Commit:
Last minute 1803 feedback

mattbriggs committed:
adding review files

Brenduns committed:
Updating date for 1803 publication

ktoliver committed:
Merge pull request #36377 from jeffgilb/1803.2

Last minute 1803 feedback

PRMerger13 committed:
Merge pull request #36405 from Brenduns/release-1803

Updating date for 1803 publication

Commit:
Final 1803 updates

PRMerger10 committed:
Merge pull request #36416 from jeffgilb/1803.3

Final 1803 updates

paulathurman committed:
edit pass: concepts-pricing-tiers

paulathurman committed:
edit pass: concepts-pricing-tiers

Brenduns committed:
New feature entry for Rotate Secrets

PRMerger12 committed:
Merge pull request #36484 from Brenduns/release-1803

New feature entry for Rotate Secrets

v-alje committed:
Resolving merge conflict

v-alje committed:
Bringing even with master.

Brenduns committed:
adding long standing known issue, and new issue for 1803

Brenduns committed:
adding long standing known issue, and new issue for 1803

Brenduns committed:
adding long standing known issue, and new issue for 1803

PRMerger10 committed:
Merge pull request #36503 from Brenduns/release-1803

adding long standing known issue, and new issue for 1803

Brenduns committed:
adding reference number to VMSS issue

PRMerger16 committed:
Merge pull request #36511 from Brenduns/release-1803

adding reference number to VMSS issue

v-alje committed:
Bringing even with master.

Brenduns committed:
Update to ASDK and IS offer process for new change state behavior

Brenduns committed:
Update to ASDK and IS offer process for new change state behavior

Brenduns committed:
New build. New number

GitHubber17 committed:
Merge pull request #36588 from Brenduns/release-1803

Update to ASDK and IS offer process for new change state behavior

v-alje committed:
Bringing even with master.

mattbriggs committed:
updating table, clearing up filenames

mattbriggs committed:
updating top level topic; adding reference topic

mattbriggs committed:
adding updated files

jaredr80 committed:
Standard Merged Peering Support

v-alje committed:
Bringing even with master.

kevinvngo committed:
Updated geo-restore capabilities

kadimitr committed:
Adding clarification for the use of sequenceNumber field, removing a sentence and modifying a link text.

mrbullwinkle committed:
Add .NET Core instructions

mattbriggs committed:
remove line from az register

v-alje committed:
Bringing even with master.

mattbriggs committed:
clearing up links

mattbriggs committed:
update profiles topic and GO topic

rachel-msft committed:
georestore portal added

rachel-msft committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into georestoreportal

v-alje committed:
Bringing even with master.

ktoliver committed:
Merge pull request #36640 from mattbriggs/mdb_pp_update

updating table, clearing up filenames

mattbriggs committed:
removing blocking issues

curtand committed:
[Azure AD groups] adding dynamic group conversion in Azure portal

gsacavdm committed:
Update Notifications Hubs

v-geberr committed:
edits

tfitzmac committed:
changed cloud shell deploy to blob

mattbriggs committed:
update numnbering

tfitzmac committed:
fixed intro

mattbriggs committed:
update

v-geberr committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 0330-edits-8

v-geberr committed:
clean up edge of images

tfitzmac committed:
changed ui elements

mattbriggs committed:
adding files

gsacavdm committed:
More fixes for gov-iothub

anrothMSFT committed:
fixing product name

chetansriv committed:
Tutorial FluxxLabs

rwike77 committed:
updated version numbers

Commit:
Merge pull request #36774 from anrothMSFT/patch-3

fixing product name

tfosmark committed:
Merge pull request #36225 from gled4er/kaniodimitrov/durable-functions-app-insights-ordering

Adding ordering for Application Insights for Durable Functions

ktoliver committed:
Merge pull request #36748 from gsacavdm/patch-2

Update Notifications Hubs

Commit:
Merge pull request #36778 from rwike77/freshness

updated version numbers

mattbriggs committed:
clear up a date that exists in alternate calender

Commit:
audit02

ktoliver committed:
Merge pull request #36768 from tfitzmac/0330ui

changed ui elements

ktoliver committed:
Merge pull request #36771 from v-geberr/0330-edits-8

0330 edits 8

rmca14 committed:
Merge pull request #35263 from mattbriggs/mdb_ProfilesSDK-1

Use Profiles for Azure Stack

rmca14 committed:
Merge pull request #36756 from tfitzmac/0330cloudshell

changed cloud shell deploy to blob

Commit:
audit02

cynthn committed:
Update virtual-machines-common-sizes-general.md

paulathurman committed:
edit pass: concepts-pricing-tiers

Commit:
audit02

tfitzmac committed:
added supported service

cherylmc committed:
table

cherylmc committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into devices3

tfitzmac committed:
updated link

karthikananth committed:
Update expressroute-locations-providers.md

PRMerger17 committed:
Merge pull request #36790 from karthikananth/patch-29

Update expressroute-locations-providers.md

ktoliver committed:
Merge pull request #36785 from tfitzmac/0330overview

added supported service

ktoliver committed:
Merge pull request #36784 from cynthn/patch-72

Update virtual-machines-common-sizes-general.md

v-alje committed:
Bringing even with master.

rmca14 committed:
Merge pull request #36789 from cherylmc/devices3

Devices3

ShawnJackson committed:
Merge pull request #36304 from paulathurman/concepts-pricing-tiers

edit pass: Concepts pricing tiers

rboucher committed:
combine all the improvements in one place

Commit:
added new FAQ

dkershaw10 committed:
Updates to AAD Graph topics

MGoedtel committed:
Updated Agent Manage article

Commit:
Merge pull request #36795 from v-payc/updatingFAQs

added new FAQ

MGoedtel committed:
minor edits

PRMerger12 committed:
Merge pull request #36797 from MGoedtel/UpdateAgentManage330

Updated Agent Manage article

dkershaw10 committed:
Updates to AAD Graph topics

PRMerger16 committed:
Merge pull request #35885 from SHEETHALJS/patch-4

(ForumsTeam)-broken links

Commit:
Merge pull request #36708 from mrbullwinkle/mrb_03_29_2018_live_stream

Add .NET Core instructions

mrbullwinkle committed:
Clarify resource selection

Commit:
Merge pull request #36800 from mrbullwinkle/mrb_03_31_2018_console

Clarify resource selection

rachel-msft committed:
Update howto-restore-server-portal.md

rachel-msft committed:
Update howto-restore-server-portal.md

rachel-msft committed:
pictures

rachel-msft committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into georestoreportal

rachel-msft committed:
date change

rachel-msft committed:
Merge branch 'georestoreportal' of https://github.com/rachel-msft/azure-docs-pr into georestoreportal

rboucher committed:
bad indent and switch order in a sentence

PRMerger17 committed:
Merge pull request #36793 from rboucher/patch-4

combine all the improvements in one place

swikrutibose committed:
(ForumsTeam)Broken Link

There is a broken link for "Naming Convention" in the section of "Create a Virtual network". I have updated that link with this: https://docs.microsoft.com/en-us/azure/architecture/best-practices/naming-conventions#naming-rules-and-restrictions. Could you please review and update the changes?

PRMerger15 committed:
Merge pull request #36667 from kevinvngo/patch-21

Updated geo-restore capabilities

PRMerger13 committed:
Merge pull request #36343 from v-mavick/free-azure-link

Corrected broken link to Free Azure subscription page.

gsacavdm committed:
Fix typo in usgov locations

PRMerger17 committed:
Merge pull request #36806 from gsacavdm/patch-1

Fix typo in usgov locations

PRMerger15 committed:
Merge pull request #36657 from jaredr80/er-merged-standard

Standard Merged Peering Support

Commit:
April updates

Commit:
Link

GitHubber17 committed:
Merge pull request #36740 from rachel-msft/georestoreportal

Georestoreportal

GitHubber17 committed:
correct ms.date year to 2018

GitHubber17 committed:
Merge pull request #36786 from chetansriv/fluxxlabs

Tutorial FluxxLabs

GitHubber17 committed:
Merge pull request #36796 from dkershaw10/djk-aadgraph-fixes

Updates to AAD Graph topics

GitHubber17 committed:
Merge pull request #36807 from cynthn/april-cpu

April updates- CPU spectre etc

mrbullwinkle committed:
SharePoint updates

mrbullwinkle committed:
sharepoint 001

GitHubber17 committed:
Merge pull request #36808 from mrbullwinkle/mrb_03_30_2018_sharepoint

SharePoint updates

Commit:
Merge pull request #1 from MicrosoftDocs/master

merge from microsoftdocs

Commit:
update function tutorial

Commit:
minor updates

Commit:
minor

PRMerger10 committed:
Merge pull request #36813 from shizn/master

update function tutorial

Sogup committed:
Update backup-azure-troubleshoot-vm-backup-fails-snapshot-timeout.md

Sogup committed:
Update backup-azure-troubleshoot-vm-backup-fails-snapshot-timeout.md

Commit:
Merge pull request #36815 from Sogup/patch-45

Update backup-azure-troubleshoot-vm-backup-fails-snapshot-timeout.md

PRMerger12 committed:
Merge pull request #36816 from Sogup/patch-46

Update backup-azure-troubleshoot-vm-backup-fails-snapshot-timeout.md

vhorne committed:
fix metrics intro

mrbullwinkle committed:
final changes

Commit:
Merge pull request #36818 from vhorne/issue-6579

fix metrics intro

mrbullwinkle committed:
pricing docs

itechedit committed:
Update active-directory-reporting-activity-audit-logs.md

billmath committed:
fixing toc'

Commit:
Merge pull request #36804 from swikrutibose/patch-8

(ForumsTeam)Broken Link

itechedit committed:
Merge pull request #35894 from mrbullwinkle/mrb_03_21_2018_pricing

App Insights Pricing Page

itechedit committed:
Merge pull request #36780 from MarkusVi/audit02

audit02

ktoliver committed:
Merge pull request #36820 from billmath/pim4

fixing toc

itechedit committed:
Merge pull request #36819 from mrbullwinkle/mrb_04_02_2018_pricing

pricing docs for April 2 publish

mmacy committed:
[ACI] portal quickstart screenshots

Commit:
Updated How Azure selects route

Commit:
Merge pull request #36823 from jimdial/patch-124

Updated How Azure selects route

CarlRabeler committed:
Update sql-database-features.md

mmacy committed:
[ACI] match UI label text

PRMerger12 committed:
Merge pull request #36824 from CarlRabeler/patch-40

Update sql-database-features.md

itechedit committed:
Merge pull request #36821 from mmacy/aci-portal-screenshots

[ACI] portal quickstart screenshots

v-alje committed:
Merge pull request #36234 from MicrosoftDocs/release-azs-1803

Release azs 1803

curtand committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into groups0329

curtand committed:
fix typo

PRMerger15 committed:
Merge pull request #36825 from mmacy/aci-portal-text-sync

[ACI] match UI label text

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

ktoliver committed:
Merge pull request #36747 from curtand/groups0329

[Azure AD groups] adding dynamic group conversion in Azure portal

v-alje committed:
Merge pull request #36826 from MicrosoftDocs/FromPublicRepo

From public repo
